README.md

```diff
@@ -8,6 +8,13 @@
 ## Hello, Finch
 <!-- markdownlint-restore -->
 
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/runfinch/finch)](https://pkg.go.dev/github.com/runfinch/finch)
+[![Go Report Card](https://goreportcard.com/badge/github.com/runfinch/finch)](https://goreportcard.com/report/github.com/runfinch/finch)
+[![CI](https://github.com/runfinch/finch/actions/workflows/ci.yaml/badge.svg?branch=main)](https://github.com/runfinch/finch/actions/workflows/ci.yaml)
+[![Static Badge](https://img.shields.io/badge/Website-Benchmarks-blue)](https://runfinch.github.io/finch/dev/bench/)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/runfinch/finch?logo=GitHub)](https://github.com/runfinch/finch/releases)
+![GitHub all releases](https://img.shields.io/github/downloads/runfinch/finch/total?label=all%20time%20downloads)
```

Comment:
nit if you want the "all time downloads" badge to be a link as well
```suggestion
[![GitHub all releases](https://img.shields.io/github/downloads/runfinch/finch/total?label=all%20time%20downloads)](https://img.shields.io/github/downloads/runfinch/finch/total?label=all%20time%20downloads)
```

Reply:
That would just link to the badge. Perhaps I can have the link go to https://github.com/runfinch/finch/releases?

Reply:
I don't think it's necessarily an issue to have two links going to the same place, and I kinda like the flow of "wow, it's been installed a lot, how can I install it?" -> the release page. +1 in favor of linking to https://github.com/runfinch/finch/releases

Reply:
Makes sense. Updated to link to releases. PTAL.